hw/ip/pinmux/pinmux_component.core

```diff
@@ -12,6 +12,7 @@ filesets:
       - lowrisc:prim:all
       - lowrisc:prim:clock_buf
       - lowrisc:prim:buf
+      - lowrisc:prim:lc_dec
```

Comment:
right, this just happened to work since the top already includes the pad stuff.

the package has a dedicated core file `lowrisc:prim:pad_wrapper_pkg`, so I'll include that here.